File: src/ProblemSolvingDayByDay/CodeForces/FridayRevise/Watermelon.java
package ProblemSolvingDayByDay.CodeForces.FridayRevise;

import java.util.Scanner;

public class Watermelon {
    public static void main(String[] args) {
        int w;
        Scanner sc = new Scanner(System.in);
        w = sc.nextInt();

        if (w % 2 != 0) {
            System.out.println("NO");
        } else {
            if (w == 2) {
                System.out.println("NO");
            } else {
                System.out.println("YES");
            }
        }


      /*  if (w == 2) {
            System.out.println("NO");
        } else if (w % 2 == 0) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }*/
    }
}

File: src/ProblemSolvingDayByDay/CodeForces/Kotlin/WaterMelon.kt
package ProblemSolvingDayByDay.CodeForces.Kotlin

/**
 * Water melon A - 1
 */
import java.util.*

fun main() {
    var w: Int
    var sc = Scanner(System.`in`)
    w = sc.nextInt()

    when {
        w == 2 -> println("No")
        w % 2 == 0 -> {
            println("Yes")
        }
        else -> {
            println("No")
        }
    }


}